cellmlmanip/model.py

```diff
@@ -232,10 +245,12 @@ def get_rdf_value(self, subject, predicate):
 
         Note: expects exactly one triple to match and the result to be a literal. Its string value is returned.
         """
-        triples = list(self.get_rdf_annotations(subject, predicate))
-        assert len(triples) == 1
-        assert isinstance(triples[0][2], rdflib.Literal)
-        value = str(triples[0][2]).strip()  # Could make this cleverer by considering data type if desired
+        triples = self.get_rdf_annotations(subject, predicate)
+        triple = next(triples, None)
+        assert triple is not None, 'Expecting exactly 1 triple'
+        assert next(triples, None) is None, "Expecting exactly 1 triple"
+        assert isinstance(triple[2], rdflib.Literal)
+        value = str(triple[2]).strip()  # Could make this cleverer by considering data type if desired
```

Comment:
This change is presumably such a minor speed optimisation that I'd be inclined to leave it as it was, since that was clearer.

Reply:
hhm ok